src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -91,6 +91,11 @@ AsyncRPCOperation_sendmany::AsyncRPCOperation_sendmany(
             throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, string("runtime error: ") + e.what());
         }
     }
+
+    // Log the context info i.e. the call parameters to z_sendmany
+    if (LogAcceptCategory("zrpcunsafe")) {
+        LogPrint("zrpcunsafe", "%s: starting (%s)\n", getId(), json_spirit::write_string( contextInfo, false));
```

Comment:
Request for change: Here's where it would be good to add the method name `z_sendmany`. In the future when we have other long operations, log parsers can distinguish between them.